docs/basic-features/eslint.md

```diff
@@ -190,11 +190,14 @@ npm install --save-dev eslint-config-prettier
 yarn add --dev eslint-config-prettier
 ```
 
-Then, add `prettier` to your existing ESLint config:
+Then, add `prettier` and the following `rules` to your existing ESLint config:
 
 ```json
 {
-  "extends": ["next", "prettier"]
+  "extends": ["next", "prettier"],
+  "rules": {
+    "indent": "error"
```

Comment:
This is incorrect as per https://github.com/prettier/eslint-config-prettier#cli-helper-tool

> While the "prettier" config can disable problematic rules in "some-other-config-you-use", it cannot touch "rules"! (That’s how ESLint works – it lets you override configs you extend.) The CLI helper tool reports that "indent" conflicts with Prettier, so you can remove it. (Which is nice – simplifying your config!)

TLDR: `"indent": "error"` is an example of a rule that should be removed according to the readme.

Reply:
Yeah you're completely correct. Is it worth mentioning the inclusion of https://github.com/prettier/eslint-plugin-prettier and config `  "plugins": ["prettier"],
  "rules": {
    "prettier/prettier": "error"
  }`

Again it's outside of this projects scope but equally a common setup with prettier